src/submission-handlers/testOrderHandler.ts

```diff
@@ -0,0 +1,61 @@
+import { map } from 'rxjs/operators';
+import { SubmissionHandler } from '..';
+import { OpenmrsEncounter, OHRIFormField } from '../api/types';
+import { EncounterContext } from '../ohri-form-context';
+
+export const TestOrderHandler: SubmissionHandler = {
+  handleFieldSubmission: (field: OHRIFormField, value: any, context: EncounterContext) => {
+    const constructNewOrder = {
+      action: 'new',
+      urgency: 'ROUTINE',
+      patient: context.patient?.id,
+      concept: value,
+      type: field?.questionOptions?.orderType,
+      careSetting: field?.questionOptions?.orderSettingUuid,
+      orderer: context?.encounterProvider,
+      encounter: context?.encounter?.uuid
+    };
+  
+    if (context.sessionMode === 'edit') {
+      const voidedPreviousOrders = context.encounter.orders?.map((element) => ({
+        uuid: element.uuid,
+        voided: true
+      }));
+      voidedPreviousOrders.push(constructNewOrder);
+      context.setOrders(voidedPreviousOrders);
+      return voidedPreviousOrders;
```

Comment:
Incase of an edit, I don't see where you link the previous order (the candidate for voiding) Is this something that happens elsewhere in the framework?